libtransmission/bitfield.cc

```diff
@@ -13,12 +13,13 @@
 #include "bitfield.h"
 #include "tr-assert.h"
 #include "utils.h" /* tr_new0() */
+#include "span.h"
 
 /****
 *****
 ****/
 
-static constexpr int8_t const trueBitCount[256] = {
+const std::array<int8_t, 256> Bitfield::true_bits_lookup_ = {
```

Comment:
I wonder if it would make any difference in the generated code if this were `constexpr`?